packages/zoe/src/contracts/exported.js

```diff
@@ -140,21 +140,20 @@
 /**
  * @typedef {Object} OraclePublicFacet the public methods accessible from the
  * contract instance
- * @property {(query: any) => ERef<Invitation>} makeQueryInvitation create an
```

Comment:
This is the only thing that seems out-of-place.  Why restrict access to make paid queries, but allow wholesale access to free queries?

Is there some security aspect that I don't understand?

Reply:
Talked with @katelynsills, and then I reinstated `publicFacet.makeQueryInvitation`: we need it for programmatic consumption of paid queries, such as with a subscription pool.